app/controllers/api/member_services_controller.rb

```diff
@@ -24,12 +24,14 @@ def gocardless_customers
   end
 
   def update_member
-    @permitted_params ||= params.permit(:email, :first_name, :last_name, :country, :postal)
-    @member_service = MemberServicesMemberService.new(@permitted_params.to_h)
-    if @member_service.update
-      render 'api/member_services/member', status: 200
+    email = params.require(:email)
+    member_params = params.require(:member).permit(:email, :first_name, :last_name, :country, :postal)
```

Comment:
I was debugging the lambda invocations from the DynamoDB table and while doing that I ran into this error:
```Jan 17 02:57:20 env-staging production.log:  I, [2018-01-17T10:57:20.811560 #5]  INFO -- : method=PUT path=/api/member_services/members format=html controller=Api::MemberServicesController action=update_member status=400 error='ActionController::ParameterMissing: param is missing or the value is empty: member' duration=0.56 view=0.00 db=0.00 params={"email"=>"tuuli@sumofus.org", "first_name"=>"Tuuli", "last_name"=>"Pollanen", "country"=>"Slovenia", "member_service"=>{"email"=>"tuuli@sumofus.org", "first_name"=>"Tuuli", "last_name"=>"Pollanen", "country"=>"Slovenia"}} time=2018-01-17 10:57:20 +0000 exception=["ActionController::ParameterMissing", "param is missing or the value is empty: member"]
```

Reply:
Sorry! I probably should've deployed development back to staging after finishing with my work.

The behaviour you see is the expected one. `email` is a required param so if it's missing a 400 Bad Request is returned, and the error gets logged.

Reply:
It was missing the `member` param but I see it's been added in your front end PR :+1: